dec/decode.go

```diff
@@ -169,10 +178,11 @@ func NewBrotliReader(stream io.Reader) *BrotliReader {
 func NewBrotliReaderSize(stream io.Reader, size int) *BrotliReader {
 	r := &BrotliReader{
 		reader: stream,
+		state:  new(C.BrotliState),
```

Comment:
I would just wait and see what people say on the forums.

The example from the youtube repo uses the []byte method, but the [Go programming book written by one of the Go developers](https://github.com/adonovan/gopl.io/blob/master/ch13/bzip/bzip2.go) uses the method you have now.